.rustfmt.toml

```diff
@@ -0,0 +1,14 @@
+# rust-analyzer can be configured with nightly rustfmt
+# i.e. for vscode in settings.json:
+#    "rust-analyzer.rustfmt.extraArgs": [
+#        "+nightly"
+#    ],
+unstable_features = true
```

Comment:
Do we really want to enable `unstable_features` in rustfmt? My worry is only that a compliant code might become noncompliant with another nightly release as unstable features are added and removed. But perhaps that's a misguided concern?

Reply:
A lot of really useful things are unavailable in stable. Pretty much every setting is unstable, but I've used them for quite some time and can't really remember stumbling into false positives or deprecated features.

Reply:
There's one macro in `aries-vcx` that's been bothering me with the formatting, but that was happening even before this PR (which points to a problem in `rustfmt` itself rather than the unstable feature). It's the `get_attach_as_string` which I decided to mark as `#[rustfmt::skip]` to avoid having to deal with it every time that file gets formatted/saved.

It's probably because `rustfmt` does not handle `let ... else {}` blocks (which are fairly new) that well right now, perhaps in particular when within a macro's body.

Reply:
Alright, let's try it, we can always easily change this setting in case we do run into issues.